ghanavinodhini/ShoppingListApp
# Jb list item screen update01

Added functionality to close the "Add new item card view" 
Added functionality to update quantity type(Eg: KG, grams) of the items.

Test Scripts:
1. Login and navigate to the shopping list item screen.
2. Click on the cart button on the right corner to add new item. Enter the details and add the item.
3. Click on the close button in the card view to close. Cart button appears after closing the card, to add new item again.
4. Edit and delete the newly added item.
